src/compiler/backend.cc

```diff
@@ -56,6 +56,7 @@ class BackendCollector : public ir::TraversingVisitor {
         case ir::Class::MONITOR:
           class_usage_counts_[klass]++;
           break;
+        case ir::Class::MIXIN:
```

Comment:
It isn't entirely clear to me why you count this as an interface here. Maybe add a comment?

Reply:
done.